packages/firestore/src/definitions.ts

```diff
@@ -377,10 +377,13 @@ export interface SnapshotListenerOptions {
    * @default false
    */
   readonly includeMetadataChanges?: boolean;
+
   /**
    * Set the source the query listens to.
    * The source `default` listens to both cache and server.
    *
+   * Note that this option is only available on the web. It is ignored on Android or iOS.
```

Comment:
```suggestion
   * Only available for Web.
```

Reply:
I've kept the "ignored on Android and iOS" part.

Reply:
Please adopt my suggestion exactly. This text is used uniformly across all plugins.